commit 76deb656132bb8817ddae4b7f417930c4db824c9
Author: Ján Tomko <jtomko@redhat.com>
Date:   Thu Jan 20 14:53:33 2022 +0100

    qemu: fix inactive snapshot revert
    
    The commit splitting out the qemuSnapshotRevertInactive function
    dropped the 'defined = true' line by accident and instead
    returned -1, leaving the user with a cryptic error:
    error: An error occurred, but the cause is unknown
    
    https://bugzilla.redhat.com/show_bug.cgi?id=2039136
    https://gitlab.com/libvirt/libvirt/-/issues/266
    
    Fixes: 85e4a13c3f19078fb6af5ffb4a80022c142cbc7e
    Signed-off-by: Ján Tomko <jtomko@redhat.com>

Index: libvirt-8.0.0/src/qemu/qemu_snapshot.c
===================================================================
--- libvirt-8.0.0.orig/src/qemu/qemu_snapshot.c
+++ libvirt-8.0.0/src/qemu/qemu_snapshot.c
@@ -2193,7 +2193,7 @@ qemuSnapshotRevertInactive(virDomainObj
 
     if (*inactiveConfig) {
         virDomainObjAssignDef(vm, inactiveConfig, false, NULL);
-        return -1;
+        defined = true;
     }
 
     if (flags & (VIR_DOMAIN_SNAPSHOT_REVERT_RUNNING |
